src/main/resources/mesosphere/marathon/api/v2/AppsResource_index.md

```diff
@@ -122,6 +125,177 @@ Transfer-Encoding: chunked
 }
 ```
 
+## GET `/v2/apps?id={identifier}`
+
+List all running applications, filtered by `id`.
+Note: the specified id must be contained in the applications id (substring). 
+
+### Example
+
+**Request:**
+
+```
+GET /v2/apps?id=my HTTP/1.1
+Accept: application/json
+Accept-Encoding: gzip, deflate, compress
+Content-Type: application/json; charset=utf-8
+Host: localhost:8080
+User-Agent: HTTPie/0.7.2
+
+
+```
+
+**Response:**
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Server: Jetty(8.y.z-SNAPSHOT)
+Transfer-Encoding: chunked
+
+{
+    "apps": [
+        {
+            "id": "/product/us-east/service/myapp", 
+            "cmd": "env && sleep 60", 
+            "constraints": [
+                [
+                    "hostname", 
+                    "UNIQUE", 
+                    ""
+                ]
+            ], 
+            "container": null, 
+            "cpus": 0.1, 
+            "env": {
+                "LD_LIBRARY_PATH": "/usr/local/lib/myLib"
+            }, 
+            "executor": "", 
+            "instances": 3, 
+            "mem": 5.0, 
+            "ports": [
+                15092, 
+                14566
+            ], 
+            "tasksRunning": 1, 
+            "tasksStaged": 0, 
+            "uris": [
+                "https://raw.github.com/mesosphere/marathon/master/README.md"
+            ], 
+            "version": "2014-03-01T23:42:20.938Z"
+        }
+    ]
+}
+```
+
+## GET `/v2/apps?label={labelSelectorQuery}`
```

Comment:
BTW, I missed that your label, cmd, id queries are in the example section and you documented that you can combine them above. That's fine.